### PR TITLE
[PLT-6623] Tests for AccordionSummary

### PR DESCRIPTION
**Pull request by imogenem**
## Description
Tests for AccordionSummary

## Notes
Warning for onFocusVisible appears but all tests pass. Once Accordion is merged with master, should be able to run without the warning.

## Where to Start
AccordionSummary.test.jsx

### CODE REVIEW

File: packages/riipen-ui/src/components/AccordionSummary.test.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  describe("icon prop", () => {
+    it("renders icon given valid custom icon prop", () => {
+      const icon = Icon;
+
+      const wrapper = mount(<AccordionSummary icon={icon} />);
+
+      expect(wrapper.find(".icon").type()).toEqual("div");
+    });
+  });
```
Comment (licarm) [inline packages/riipen-ui/src/components/AccordionSummary.test.jsx:71]: You can assert that the provided icon is being rendered using the assertion .containsMatchingElement() or .contains().

**Review by licarm — CHANGES_REQUESTED** [review]
Nice!! Just one thing.

File: packages/riipen-ui/src/components/AccordionSummary.test.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+      const wrapper = mount(<AccordionSummary icon={icon} />);
+
+      expect(
+        wrapper.find("AccordionSummary").containsMatchingElement(icon)
+      ).toEqual(true);
+    });
+  });
+
+  describe("iconProps prop", () => {
```
Comment (jordanell) [inline packages/riipen-ui/src/components/AccordionSummary.test.jsx:75]: I think we should add another test here. The `iconProps` prop also looks like it passes `iconProps.classes` to the icon `classes` here https://github.com/riipen/ui/blob/eed21d980e392b67816338eb5cfc816c5e877746/packages/riipen-ui/src/components/AccordionSummary.jsx#L94. 

So let's add a test to make sure that happens.

**Review by jordanell — CHANGES_REQUESTED** [review]
Nice job!

**Review by jordanell — APPROVED** [review]

File: packages/riipen-ui/src/components/AccordionSummary.test.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      const icon = Icon;
+      const iconProps = { classes: ["small"] };
+
+      const wrapper = mount(
+        <AccordionSummary icon={icon} iconProps={iconProps} />
+      );
+
+      expect(wrapper.find("ButtonIcon").props().classes[0]).toContain(
+        iconProps.classes
+      );
```
Comment (licarm) [inline packages/riipen-ui/src/components/AccordionSummary.test.jsx:97]: Look great!

**Review by licarm — APPROVED** [review]

**Pull request merged**